lib/web/ui/usercontext.go

```diff
@@ -104,7 +104,8 @@ func NewUserContext(user types.User, userRoles services.RoleSet, features proto.
 	authType := authLocal
 
 	// check for any SSO identities
-	isSSO := len(user.GetOIDCIdentities()) > 0 ||
+	isSSO := user.GetUserType() == types.UserTypeSSO ||
+		len(user.GetOIDCIdentities()) > 0 ||
```

Comment:
looks good to me. just curious if `user.GetUserType()` is enough without check other identities?

Reply:
It only checks for the `CreatedBy.Connector != nil`.
I did that, but some tests started failing so I'm not sure we should be changing that.

On the other hard, maybe `user.GetUserType()` should check not only for the `CreatedBy.Connector` but also for the OIDC and SAML identities.